runtime/wasm/src/module/mod.rs

```diff
@@ -840,7 +843,28 @@ impl<C: Blockchain> WasmInstanceContext<C> {
                     .start_section("store_get_asc_new");
                 asc_new(self, &entity.sorted())?
             }
-            None => AscPtr::null(),
+            None => match &self.ctx.debug_fork {
+                Some(fork) => {
+                    let entity = fork.fetch(entity_type, id).map_err(|e| {
+                        // TODO: Better error handling.
+                        HostExportError::Unknown(anyhow!("Failed to fetch entity from fork: {}", e))
+                    })?;
+                    match entity {
+                        Some(entity) => {
+                            // TODO: Probably change the name of the section?
+                            let _section = self
+                                .host_metrics
+                                .stopwatch
+                                .start_section("store_get_asc_new");
```

Comment:
Personally I would change this section to `store_get_debug_fork__asc_new`, perhaps @leoyvens or @Jannis can give a suggestion.

Reply:
imo we can keep the same section name, probably nobody is going to benchmark graph node with a fork anyways.